build-system/tasks/presubmit-checks.js

```diff
@@ -414,6 +414,8 @@ var forbiddenTermsSrcInclusive = {
   '\\.webkitConvertPointFromPageToNode(?!_)': bannedTermsHelpString,
   '\\.changeHeight(?!_)': bannedTermsHelpString,
   '\\.changeSize(?!_)': bannedTermsHelpString,
+  'insertAmpExtensionScript(?!\\s*\/\\*OK\\*\\/)(?!\\s*\/\\*REVIEW\\*\\/)':
```

Comment:
Very specific :)

REVIEW should not be whitelisted without any existing usages.